mappings/net/minecraft/entity/passive/StriderEntity.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_4985 net/minecraft/entity/passive/StriderEntity
+	FIELD field_23240 saddledComponent Lnet/minecraft/class_4980;
+	FIELD field_23243 BREEDING_INGREDIENT Lnet/minecraft/class_1856;
+	FIELD field_23244 ATTRACTING_INGREDIENT Lnet/minecraft/class_1856;
+	FIELD field_23247 SADDLED Lnet/minecraft/class_2940;
+	METHOD method_26344 canSpawn (Lnet/minecraft/class_1299;Lnet/minecraft/class_1936;Lnet/minecraft/class_3730;Lnet/minecraft/class_2338;Ljava/util/Random;)Z
+		ARG 0 type
+		ARG 1 world
+		ARG 2 spawnType
+		ARG 3 pos
+		ARG 4 random
+	CLASS class_4986 StriderData
+		CLASS class_4987 RiderType
```

Comment:
Please map `class_4988` as well, should be `Navigation`

Reply:
done